tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -304,7 +308,58 @@ def create_fsx_open_zfs(fsx_factory, num):
     volume_list = boto3.client("fsx").describe_volumes(Filters=[{"Name": "file-system-id", "Values": file_system_ids}])[
         "Volumes"
     ]
-    return [(volume["FileSystemId"], volume["VolumeId"]) for volume in volume_list]
+    return [volume["VolumeId"] for volume in volume_list]
+
+
+@pytest.fixture(scope="class")
+def open_zfs_volume_factory(vpc_stack, cfn_stacks_factory, request, region, key_name):
+    """
+    Define a fixture to manage the creation and destruction of volumes for FSx for OpenZFS.
+
+    return volume ids
+    """
+    created_stacks = []
+
+    def _open_zfs_volume_factory(root_volume_id, num_volumes=1):
+        fsx_open_zfs_volume_template = Template()
+        fsx_open_zfs_volume_template.set_version()
+        fsx_open_zfs_volume_template.set_description("Create Storage Virtual Machine stack")
+
+        open_zfs_volume_resource_name = "OpenZFSVolume"
+        max_concurrency = 15
```

Comment:
What do we mean here with concurrency?

Reply:
When create multiple file systems, CloudFormation creates them concurrently. To avoid throttling, we need to limit the concurrency